test/module-registry.spec.js

```diff
@@ -19,6 +19,17 @@ describe('Module Registry.', () => {
     expect(result).to.be.an.instanceOf(MyModule);
   });
 
+  it('should be able to pass parameters to the register a module', () => {
+    class MyModule {
+      constructor(name) {
+        this.name = name;
+      }
+    }
+    ModuleRegistry.registerModule('GLOBAL_ID', MyModule, 'DUMMY_NAME');
```

Comment:
may be it's better to have the following signature:
`registerModule('Global_ID', MyModules, ["DUMMY_NAME"]);`
Wrap arguments as an array. In case you'd like to pass something as 4th parameter.

Reply:
it's es6's `...args` it's an array